common/views/components/ViewerTopBar/ViewerTopBar.tsx

```diff
@@ -98,8 +98,7 @@ const ViewAllContainer = styled.div.attrs(() => ({
 }))`
   height: 64px;
   width: 20%;
-  border-right: 1px solid
-    ${props => lighten(0.1, props.theme.color('viewerBlack'))};
+  border-right: 10px solid red;
```

Comment:
?

Reply:
haha! good spot. I'm in the process of deleting this file in another pr as it's not in use, but I'll undo this change.